### ISSUE TITLE
Issue with tutorials/run-an-elixir-phoenix-app-in-containers-using-google-kubernetes-engine.md

### ISSUE THREAD

**Issue by docJerem** [GoogleCloudPlatform/community#1979]
## nodejs-npm is deprecated

<img width="719" alt="image" src="https://user-images.githubusercontent.com/10330097/142191870-10c77bbc-d1c2-41d0-ba4d-99d444e81f8a.png">

## Solution

In the Dockerfile you need to rename

```
    && apk --no-cache --update add nodejs nodejs-npm \
```

to 

```
&& apk --no-cache --update add nodejs npm \
```

**Comment by ToddKopriva** [comment]
Thanks, @docJerem . If you want to create a pull request to make this change yourself, we'll review it.

@dazuma , can you take a look at this recommendation for a document that you authored?

**Comment by ToddKopriva** [comment]
fixed with PR #2160
